src/main/java/roomescape/reservation/domain/Reservation.java

```diff
@@ -23,12 +23,11 @@ public class Reservation {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
+    private LocalDate date;
```

Comment:
빈 줄을 추가해 가독성을 높여주세요!

```suggestion

    private LocalDate date;
```